pyracing/client.py

```diff
@@ -716,10 +774,21 @@ async def season_standings(
 
         url = ct.URL_SEASON_STANDINGS
         response = await self._build_request(url, payload)
-
-        return [historical_data.SeasonStandings(x) for
-                x in response.json()["d"]["r"]]
-
+        
+        if response.json():
+            mapping = response.json().get('m')
+            response_data = response.json().get('d')
+
+            if mapping and response_data:
```

Comment:
In some places you check `if mapping and response_data:` and some you only check `if response_items:` We should probably be as consistent as possible with what we are checking. I think other than that this is looking good.  Once the code is updated I will approve and we can merge this

Reply:
Do you mean the different names? So .get('d') would be put into response_data and .get('r') into response_items?

Reply:
No, I am saying inside a few methods like `series_race_results` you don't check `if mapping and response_data`. Should we be checking that everywhere and short circuiting if one of them is None?

Reply:
Yup, totally missed that in `﻿series_race_results`. But that was the only where the check was missing.